about.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>About</title>
+  </head>
+  <body class="background">
+    <header class="header">
+      <div class="hamburger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+      <img
+        class="company-logo-about"
+        src="images/company_logo.png"
+        alt="company-logo"
+      />
+      <nav class="main-navigation main-navigation-about">
+        <ul class="navigation-list">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Products</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">News</a></li>
```

Comment:
- [ ] Follow the original font color design.

Original design:

![image](https://user-images.githubusercontent.com/60631456/124143537-d1fa1b00-da50-11eb-8301-4d81670b5f35.png)

Your design:

![image](https://user-images.githubusercontent.com/60631456/124143849-12f22f80-da51-11eb-87c3-81a0e78b28f3.png)

Reply:
Thank you, I changed the font color